core/templates/dev/head/player/AnswerClassificationService.js

```diff
@@ -81,8 +82,7 @@ oppia.factory('answerClassificationService', [
        * @param {boolean} isInEditorMode - Whether the function is being called
        *   in editor mode.
        * @param {?function} interactionRulesService - The service which contains
-       *   the rules of that interaction. If this is undefined, then the
-       *   function uses server-side classification.
+       *   the rules of that interaction.
```

Comment:
Just to check, does this refer to 'hard' rules only? If so, might be worth adding the word 'hard' here.